src/core/icons/folderIcons.ts

```diff
@@ -79,10 +94,13 @@ export const folderIcons: FolderTheme[] = [
       },
       {
         name: 'folder-javascript',
-        folderNames: ['js', 'javascript', 'javascripts'],
+        folderNames: ['js', 'javascript', 'javascripts', 'cjs', 'mjs'],
+      },
+      { name: 'folder-json', folderNames: ['json', 'jsons', 'jsonc', 'jsonl'] },
+      {
+        name: 'folder-font',
+        folderNames: ['font', 'fonts', 'typeface', 'typesaces'],
```

Comment:
I didn't see this in the first review, but there's a typo

```suggestion
        folderNames: ['font', 'fonts', 'typeface', 'typefaces'],
```
```suggestion
        folderNames: ['font', 'fonts', 'typeface', 'typesaces'],
```